software/firmware/bootloader.py

```diff
@@ -1,18 +1,17 @@
+import europi
 import gc
-import time
-from collections import OrderedDict
-
 import machine
+import sys
+import time
 
-import europi
+from collections import OrderedDict
 from europi import (
     reset_state,
     OLED_HEIGHT,
     OLED_WIDTH,
```

Comment:
[required] `CHAR_WIDTH` is missing here, which causes an exception during error handling.

Reply:
Done